gitlab_total_repo_count

```diff
@@ -16,10 +16,13 @@ if len(sys.argv) >= 2 and sys.argv[1] == 'config':
     print('repo_count.draw AREA')
     print('wiki_count.label Wiki Count')
     print('wiki_count.draw STACK')
+    print('wiki_unused_count.label Wiki Count')
```

Comment:
Could you please add "(unused)" (or something similar) there?

Reply:
Yes, correct. As I just found "empty" as appropriate, update this.